### PR TITLE
Add Crypto Payment Widget, QR code, and assets

### PR DESCRIPTION
**Pull request by jlbyrne**
Resolves https://github.com/brave-intl/creators-private-issues/issues/1651
Resolves https://github.com/brave-intl/creators-private-issues/issues/1650
Resolves https://github.com/brave-intl/creators-private-issues/issues/1649

**Testing Instructions:**

log in as [Alice@default.org](mailto:Alice@default.org).  When landing on the publisher’s dashboard, make sure to assign some of the existing crypto addresses to the ‘uphold connected twitter’ channel.  Then you should be able to navigate to https://localhost:3000/c/c6587d7de522444595e2, and see the channel page details and widget.    Press ‘send’ to connect a wallet.  The user can select what network they’re on, so check to make sure you are on a test network.  Faucets for Solana and eth can be found here: https://solfaucet.com/, https://faucet.quicknode.com/ethereum/sepolia

### CODE REVIEW

File: app/javascript/views/publicChannelPage/SuccessWidget.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    this.state = {
+      setStateToStart: props.setStateToStart,
+      amount: props.amount,
+      chain: props.chain,
+    }
+  }
+
+  shareSupport(){
+    // ????
```
Comment (tsmartt) [inline app/javascript/views/publicChannelPage/SuccessWidget.jsx:25]: Curious what this is?

File: app/javascript/views/publicChannelPage/CryptoPaymentWidget.jsx [hunk 2]
```diff
@@ -1,20 +1,360 @@
...
+        })
+        .then((result) => {
+          // window.ethereum.disable()
+          const newState = {...this.state};
+          newState.isSuccessView = true;
+          this.setState({...newState });
+        })
+        .catch((error) => {
+          console.log(error)
+          // window.ethereum.disable()
```
Comment (tsmartt) [inline app/javascript/views/publicChannelPage/CryptoPaymentWidget.jsx:167]: Do we want to leave this in?

**Review by tsmartt — APPROVED** [review]

File: app/javascript/views/publicChannelPage/SuccessWidget.jsx (hunk 1 above)
Comment (jlbyrne) [reply app/javascript/views/publicChannelPage/SuccessWidget.jsx:25]: We don't have behavior described for the sharesupport button yet.  That's a placeholder for a future ticket where we'll add some behavior in.

File: app/javascript/views/publicChannelPage/CryptoPaymentWidget.jsx (hunk 2 above)
Comment (jlbyrne) [reply app/javascript/views/publicChannelPage/CryptoPaymentWidget.jsx:167]: We can get rid of it.  Apparently window.ethereum.disable() is spec'd out, but hasn't yet been implmented.  I'm hoping to find another way for the page to close the connection with the wallet in the future.

**Review by tsmartt — APPROVED** [review]

**Pull request merged**